samples/BenchmarkDotNet.Samples/DoNothing.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading;
```

Comment:
I created this sample project for debugging and was going to remove it, but it occured to me that it is actually a pretty useful sample to have around. I can delete it if that is preferred though.